client/src/Components/Spectrogram/SpectrogramPage.tsx

```diff
@@ -525,9 +526,15 @@ export const SpectrogramPage = () => {
                 )}
               </div>
             </div>
+            <MetaViewer
```

Comment:
I would just pass meta as the parameters and use the methods inside to grab the data they are a bit more reliable and take care or hard coded string

Reply:
yep, done